EventFilter/Phase2TrackerRawToDigi/test/SLinkProducerAndUnpacker_cfg.py

```diff
@@ -0,0 +1,137 @@
+#!/cvmfs/cms.cern.ch/el9_amd64_gcc12/cms/cmssw/CMSSW_14_0_0_pre2/bin/el9_amd64_gcc12/cmsRun
+## Stub Occupancy Script
```

Comment:
Longer comment needed to explain what this job does.

Reply:
added comment